factory/processing/processComponents_test.go

```diff
@@ -655,6 +659,67 @@ func TestNewProcessComponentsFactory(t *testing.T) {
 		require.NotNil(t, pcf)
 	})
 }
+func TestProcessComponentsFactory_AddSystemVMToContainer(t *testing.T) {
+	t.Parallel()
+
+	testContainerKey := "testContainer"
+
+	t.Run("sovereign should add systemVM", func(t *testing.T) {
+		t.Parallel()
+
+		shardCoordinator := mock.NewMultiShardsCoordinatorMock(2)
+		processArgs := components.GetProcessComponentsFactoryArgs(shardCoordinator)
+		processArgs.ChainRunType = common.ChainRunTypeSovereign
+		pcf, _ := processComp.NewProcessComponentsFactory(processArgs)
+		require.NotNil(t, pcf)
+		vms := make(map[string]vmcommon.VMExecutionHandler, 0)
+		vms[testContainerKey] = &processMock.VMExecutionHandlerStub{}
```

Comment:
Can remove this and checking for this test key

Reply:
wanted to check that the "Add" does not change the previous vms in the container